db/migrations/20190313145917_create_movies.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 exports.up = async (Knex) => {
-        await Knex.schema.createTable('movies', (table) => {
+    await Knex.schema.createTable('movies', (table) => {
```

Comment:
Looks like this file is indenting with 4 spaces. We typically indent with 2 spaces in our Javascript repositories.

Reply:
oh. i'll change the settings